xml/admin_administration.xml

```diff
@@ -205,40 +205,45 @@
   <sect2 xml:id="commands.node.managment.issue-generator">
    <title>The <command>issue-generator</command> Command</title>
    <para>
-    The <command>issue-generator</command> creates a volatile temporary
-    <filename>/run/issue</filename> file. The file
-    <filename>/etc/issue</filename> should be a symbolic link to the temporary
-    <filename>/run/issue</filename>.
+    On &suse; &mos;, <filename>/etc/issue</filename> is generated by <command>issue-generator</command>.
+   </para>
+   <para>The <command>issue-generator</command> program allows &mos; to display
+    a dynamic message above the login prompt, which is generated from the
+    contents of several different files.
    </para>
    <para>
-    You can use the command to prefix all directories and files with a
-    specified prefix (PATH in this example):
+    Files in <filename>/etc/issue.d</filename> override files with the same name in
+    <filename>/usr/lib/issue.d</filename> and <filename>/run/issue.d</filename>.
    </para>
-<screen>&prompt.root;<command>issue-generator --prefix <replaceable>PATH</replaceable></command></screen>
    <para>
-    By using the command you can also create or delete files in the network
-    configuration, for example:
+    Files in <filename>/run/issue.d</filename> override files with the same name
+    in <filename>/usr/lib/issue.d</filename>.
    </para>
-<screen>&prompt.root;<command>issue-generator network <replaceable>remove</replaceable> <replaceable>INTERFACE</replaceable></command></screen>
    <para>
-    The command removes file
-    <filename>/run/issue.d/70-<replaceable>INTERFACE</replaceable>.conf
-    </filename>. The file contains the name of the and escape codes for
-    <command>agetty</command>.
+    Packages should install their configuration files in <filename>/usr/lib/issue.d</filename>.
    </para>
    <para>
```

Comment:
Definitely no new paragraph here, these two sentences belong together. User need to understand the difference between /etc/issue.d and /usr/lib/issue.d. The latter is only for files that are shipped with packages. Changes there would be overwritten by updates, so admins must only make changes or additions in /etc/issue.d.

Reply:
Done.